.github/workflows/tests.yaml

```diff
@@ -0,0 +1,36 @@
+name: Tests
+on: [push]
+jobs:
+  tests:
+    name: ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.8, 3.9]
```

Comment:
The readme says 3.5 and above, so should it be here, too? Also, why not include 3.7?

Reply:
Let's just bump the minimum supported Python version in the README (and the package description if that's a thing in Python packages) to 3.6, then.